src/index.ts

```diff
@@ -96,28 +90,20 @@ export const plugin: Plugin = {
   },
   printers: {
     'pug-ast': {
-      print(
+      // @ts-expect-error -- Prettier allow it to be async if we don't do recursively print
+      async print(
```

Comment:
@fisker 

> Prettier allow it to be async if we don't do recursively print

So why is there no overloading like:

```ts
  print(
    path: AstPath<T>,
    options: ParserOptions<T>
  ): Doc;
```

Reply:
I can't understand what do mean.